spec/cancan/model_adapters/accessible_by_integration_spec.rb

```diff
@@ -76,7 +76,7 @@ class Editor < ActiveRecord::Base
 
   describe 'preloading of associatons' do
     it 'preloads associations correctly' do
-      posts = Post.accessible_by(ability).includes(likes: :user)
+      posts = Post.accessible_by(ability).where(published: true).includes(likes: :user)
```

Comment:
i don't think these changes had any affect on tests?

Reply:
It did. See the explanation https://github.com/CanCanCommunity/cancancan/pull/605#issuecomment-556009976

The query changed just enough to get postgres to return a different record (because the query contained 2 but we applied limit 1).

Reply:
Ahhh yep right you are. I was only testing locally in sqlite when I wrote that comment. It seemed to not make a difference there.